contributing/development/compiling/compiling_for_macos.rst

```diff
@@ -55,6 +55,9 @@ To compile for Apple Silicon (ARM64) powered Macs, use::
 
     scons platform=macos arch=arm64
 
+You may need to install MoltenVK first::
+    brew install molten-vk
```

Comment:
```suggestion

    brew install molten-vk
```

Reply:
It is already mentioned above (VulkanSDK). And in general you should not use homebrew for it, it includes both dynamic and static library, and linker will prefer dynamic library. It's OK for test builds, but with dynamic library your build will not be portable. Also, you won't be able to build for another architecture.